README.md

```diff
@@ -1,2 +1,38 @@
+<p align="center">
+  <img width="626px" height="280px" src="assets/logo.png">
+</p>
+
 # Pulsarc
-A community-created rhythm game! Development Discord: https://discord.gg/SYfpvfJ
+[![Discord](https://discordapp.com/api/guilds/486933399425122318/widget.png?style=shield)](https://discord.gg/SYfpvfJ)
+>A community-created rhythm game!
+
+## About
+[Read more about this project!](docs/ABOUT.md)
+
+## How to Play the Prototype
+When the prototype is finished we will have a public release build people can use.
+If you want to play now though, you need:
+
+**Visual Studio Community** - https://docs.microsoft.com/en-us/visualstudio/releasenotes/vs2017-relnotes
+
+**.NET Core 2.2 SDK (v2.2.108)** - https://dotnet.microsoft.com/download/thank-you/dotnet-sdk-2.2.108-windows-x64-installer
+
+**MonoGame 3.7.1 for VisualStudio** - http://community.monogame.net/t/monogame-3-7-1-release/11173
```

Comment:
Not needed since we get it through submodules

Reply:
I'm not sure if this will be added through submodules since these are installed via windows installer

Reply:
We are using monogame submodule in Wobble, which should have all monogame stuff that we are using.

Reply:
Yes but compiling and some content related features rely on you having monogame and the monogame pipeline installed on your computer. If you are not on windows it might not be necessary. I'm not sure to what extend you can do stuff without installing, but just to avoid future troubleshooting it'd be best to have all contributors and testers have those installed until we deliver some published builds